.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: []
```

actionlint (1.7.10)

[error] "branches" section should not be empty

(syntax-check)